src/docs/product/alerts/best-practices.mdx

```diff
@@ -20,8 +20,8 @@ An issue alert is triggered when an individual issue meets some criteria. These
 The following triggers, or “when” conditions, capture state changes in issues:
 
 - A new issue is created
-- The issue changes state from resolved to unresolved or a regression occurs
-- The issue changes state from ignored to unresolved — that is, an ignored issue's ignore-until conditions are met
+- The issue changes state from resolved to unresolved, or a regression occurs
+- The issue changes state from archived to escalating
```

Comment:
should we replace the explanation instead of removing it? (ie, `— that is, an archived issue's archived-until conditions are met`)

Reply:
The UI says "issue changes state from archived to escalating". I thought an issue doesn't necessarily escalate and trigger the alert when its archived-until conditions are met. For instance, if you archive for 30 mins or until the issue occurs again 10 times, the issue wouldn't necessarily be tagged as escalating. 

LMK if this isn't correct and the alert fires anyway whenever the archived-until conditions are met.

Reply:
ah yes, you're right - thank you!